doc/releases/release-notes-3.7.rst

```diff
@@ -182,6 +182,9 @@ Drivers and Sensors
 
 * Wi-Fi
 
+  * Added support to configure RTS threshold. User can set RTS threshold value or
```

Comment:
Does this mean
"Added support for configuring the RTS threshold. With this, users can set the RTS threshold value or disable the RTS mechanism"
?
(if so, could you correct this text?